integration/README.md

```diff
@@ -65,17 +65,41 @@ All config options are optional. If you don't pass any config, it will use the d
 
-#### Channel Option
+### Channel Option
 
 Channel option is used to make the iframe open chat room (a channel) directly. This is useful if you want to have a specific topic for your user to discuss.
 
-| Name       | Type                                   | Description                                                                          |
-| ---------- | -------------------------------------- | ------------------------------------------------------------------------------------ |
-| `type`     | `'channel'`                            | The type of the channel. This should be set to `'channel'`                           |
-| `id`       | `string`                               | The id of the channel. This should be the post id of the topic that you want to open |
-| `settings` | [`ChannelSettings`](#channel-settings) | The settings of the channel. Read more about this [here](#channel-settings)          |
+| Name       | Type                                   | Description                                                                                                                |
+| ---------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `type`     | `'channel'` &#124; `'resource'`        | The type of the channel. Check options for [channel](#type-channel-options) and [resource](#type-resource-options) bellow. |
+| `settings` | [`ChannelSettings`](#channel-settings) | The settings of the channel. Read more about this [here](#channel-settings)                                                |
 
-##### Channel Settings
+#### Type `'channel'` Options
+
+The type opens static chat room by id
+
+| Name | Type     | Description                                                                             |
+| ---- | -------- | --------------------------------------------------------------------------------------- |
+| `id` | `string` | The id of the channel. This should be the channel id of the topic that you want to open |
+
+#### Type `'resource'` Options
+
+The type creates new or opens existed chat room by resource.
+
+| Name       | Type                     | Description                                                                                                                                                                          |
+| ---------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `resource` | `Resource`               | The resource that will be linked to the postId in the blockchain should be the Resource. You can find examples suitable for various scenarios [here](#resource-discussion-examples). |
```

Comment:
```suggestion
| `resource` | `Resource`               | The resource linked to the postId on the blockchain should be the Resource. You can find examples suitable for various scenarios [here](#resource-discussion-examples). |
```

Reply:
The first sentence seems like... duh? Obvious? Is it needed? Or am I not understanding it correctly?